lib/parsedate.c

```diff
@@ -587,6 +587,31 @@ time_t curl_getdate(const char *p, const time_t *now)
   return -1;
 }
 
+/* Curl_getdate_capped() differs from curl_getdate() in that this will return
+   TIME_T_MAX in case the parsed time value was too big, instead of an
+   error. */
+
+time_t Curl_getdate_capped(const char *p, const time_t *now)
+{
+  time_t parsed = -1;
+  int rc = parsedate(p, &parsed);
+  (void)now; /* legacy argument from the past that we ignore */
```

Comment:
As this is a new function with no backwards compatibility baggage, can't we skip the `now` parameter?

Reply:
Oh right, we certainly should get rid of that there. Thanks!